website/docs/cli/code/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 page_title: Writing and Modifying Code - Terraform CLI
 description: >-
-  Learn commands that help validate, format, and upgrade code written in the
-  Terraform Configuration Language.
+  Learn commands that help validate, format, and upgrade code written in 
+  Terraform's configuration language.
```

Comment:
Used in different places like this, e.g.  

- https://developer.hashicorp.com/terraform/language/style
- https://developer.hashicorp.com/terraform/language/syntax/configuration

Other places use "the Terraform language".